### ISSUE TITLE
[Portworx] For group snapshots, wait till all PVCs with group ID are bound

### ISSUE THREAD

**Issue by disrani-px** [libopenstorage/stork#160]
**Is this a BUG REPORT or FEATURE REQUEST?**:

**What happened**:

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by disrani-px** [comment]
@harsh-px Is this fixed in 2.1.0?

**Comment by harsh-px** [comment]
2.1.0 deprecated group snapshots using group ID. So this issue is no longer valid.

```
	case crdv1.PortworxSnapshotTypeLocal:
		if isGroupSnap(snap) {
			err = &errors.ErrNotSupported{
				Feature: "Group snapshots using VolumeSnapshot with annotations",
				Reason:  "Since 2.0.2 group snapshots are only supported using GroupVolumeSnapshot CRD.",
			}

			return nil, getErrorSnapshotConditions(err), err
		}
```
